File: BE/Boek.Infrastructure/ViewModels/Groups/GroupViewModel.cs
using Boek.Infrastructure.Attributes;
using Boek.Infrastructure.ViewModels.Campaigns;
using Boek.Infrastructure.ViewModels.Users.Customers;

namespace Boek.Infrastructure.ViewModels.Groups
{
    public class GroupViewModel
    {
        [Int]
        public int? Id { get; set; }

        [String]
        public string Name { get; set; }

        [String]
        public string Description { get; set; }

        [Boolean]
        public bool? Status { get; set; }
        [String]
        public string StatusName { get; set; }

        public List<CustomerUserViewModel> Customers { get; set; }

        public List<BasicCampaignViewModel> Campaigns { get; set; }
    }
}

File: Mobile/objects/viewmodels/Groups/GroupViewModel.ts
import { BasicCampaignViewModel } from "../Campaigns/BasicCampaignViewModel";
import { CustomerUserViewModel } from "../Users/customers/CustomerUserViewModel";

export interface GroupViewModel {
    id: number;
    name: string;
    description: string;
    status: boolean;
    statusName: string;
    customers: CustomerUserViewModel[];
    campaigns: BasicCampaignViewModel[];
}